File: data/src/main/java/home/oleg/placesnearme/mapper/HoursMapper.java
package home.oleg.placesnearme.mapper;

import home.oleg.placenearme.models.Hours;

/**
 * Created by Oleg Sheliakin on 25.09.2018.
 * Contact me by email - olegsheliakin@gmail.com
 */
public final class HoursMapper {

    private HoursMapper() {
    }

    public static Hours map(home.oleg.placesnearme.core_network.models.Hours hours) {
        Hours result = new Hours();

        result.setIsLocalHoliday(hours.getIsLocalHoliday());
        result.setIsOpen(hours.getIsOpen());
        result.setStatus(hours.getStatus());

        return result;
    }


}

File: data/src/main/java/home/oleg/placesnearme/data/mapper/HoursMapper.kt
package home.oleg.placesnearme.data.mapper

import home.oleg.placenearme.models.Hours

/**
 * Created by Oleg Sheliakin on 25.09.2018.
 * Contact me by email - olegsheliakin@gmail.com
 */
object HoursMapper {

    fun map(hours: home.oleg.placesnearme.network.models.Hours) = Hours(
            isOpen = hours.isOpen,
            status = hours.status
    )
}
